configure.ac

```diff
@@ -27,6 +27,8 @@ AC_PROG_MKDIR_P
 AC_PATH_PROG([FIND], [find])
 AS_IF([test -z "${FIND}"], [AC_MSG_ERROR(['find' utility not found])])
 
+AC_PATH_PROG([CURL], [curl])
```

Comment:
If we get a request we could make this dependency optional in the future but for now I would just leave it like it is.